### DISCUSSION TITLE
Having trouble using the linear Programming Tutorials

### DISCUSSION THREAD

**Discussion by Cbapti** [yalmip/YALMIP#1351] [Problems using YALMIP] [answered]
Hello,

Im pretty new to YALMIP, thats why I tried some tutorials. After the getting started I tried to work my way through the linear programming Tutorial:

https://yalmip.github.io/tutorial/linearprogramming/

Unfortunately i get a error when I run the exact code from the Homepage. I get the following error message:

"Error using linprog↵LINPROG(f,A,b,Aeq,beq,LB,UB,X0,OPTIONS) does not accept X0. Use LINPROG(f,A,b,Aeq,beq,LB,UB,OPTIONS"

I only get NaN values for a and b then. 

I thought about solving this by replacing the linprog call with the suggest one from the error message but Im asking here because I think there might be a better solution or Im missing smth fundamental.

Thanks in advance for your help!

Edit:

Using Matlab R2023b Update 1 with the Optimization Toolbox installed

Best regards,

Christian

**Comment by johanlofberg** [comment]
What is the result from

 which linprog -all

**Reply by johanlofberg** [reply, marked as answer]
It looks like they've introduced a breaking change in r2023 by removing the x0 argument. Hence, you probably have to edit the file calllinprog and remove the argument x0 in the call to linprog

**Reply by Cbapti** [reply]
I did that, the error has gone. Thanks a lot for your help!
